inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.15/pom.xml

```diff
@@ -241,7 +254,7 @@
                         <goals>
                             <goal>copy</goal>
                         </goals>
-                        <phase>pre-integration-test</phase>
+                        <phase>validate</phase>
```

Comment:
sorry but why change to validate ?

Reply:
Because maven lifecycle does not have a pre-integration-test phase, and this phaseis not defined in the code, the end-to-end module cannot add the sort-connector jar to dependencies when packaging.So it need to copy dependencies to target directory before package.Some pictures as follow.

![I_H7ZUTJ40L{2GLC8 JN$PP](https://github.com/apache/inlong/assets/58425449/4bdd11d6-8138-450f-8469-bb86dfa51029)

![image](https://github.com/apache/inlong/assets/58425449/52d6ccea-b075-469a-a597-0ff56606213b)